src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms linear;
```

Comment:
add `transition-property` to show what exactly you would like to change. By default `transition-property` applies to `all`
You need only to `color` and `transform`

Reply:
ok